packages/flutter_tools/templates/package/LICENSE.tmpl

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 Your Company. All rights reserved.
```

Comment:
Yeah. Can you do that in this PR also?

We should probably just have placeholder LICENSE files so that people see that they should fill it in. e.g. just a TODO.